README.md

```diff
@@ -2,6 +2,25 @@
 
 Cleisthenes is PBFT consensus library adaptable to any application.
 
+## Architecture
+
+### Logical View
+
+<p align="center">
+	<img src="./img/cleisthenes-architecture-logical-view.png" ></img>
```

Comment:
### Between `SWIM` and `Members`, is `fetch nodes status` better than `update member status`?
I think that `SWIM` doesn't modify any data of `Members`. 
Instead, `Member` fetches the nodes status from `SWIM` and updates member status. Is it right?
If what I think is right, `update member status` might be confusing.
So, how about changing `update member status` to `fetch nodes status` and adding `update member status` into `Members`?

### I suggest using `node`, not `member` in `SWIM`.
Using the word `member` in SWIM could be confusing.
So, I suggest two things.
1. `node` means a peer in network.
2. `member` means a node who participates in consensus.

In this way, `node` is only used in `SWIM` layer and `member` is used in `Members` or `PBFT` layer.

Reply:
I agree term of `node` and `member`. 

Can you explain more about first suggestion? It sounds like that `swim` act as **only** node discovery. right?

Reply:
Yes, you're right!

Reply:
@hihiboss @AgwaB 
## update vs fetch
> I think that SWIM doesn't modify any data of Members.
Instead, Members fetches the nodes status from SWIM and updates member status. Is it right?
If what I think is right, update member status might be confusing.
So, how about changing update member status to fetch nodes status and adding update member status into Members?

I think there's two way of implementing how to communicate between `SWIM` and `Members`.

1. pulling `SWIM` member map periodically from `Members` (**`Members` fetch `SWIM` data**)
2. signal to `Members` when `SWIM` has a change on its member map (**`SWIM` updates `Members` states**)

IMO I think 2. way is better design, because it minimize communication between components.

## member & node
I think change member to node is good. 

but I disagree defining `member` as who participate in consensus. Because IT IS TOO **Indistinguishable**. 

 It would be better to have a word other than `member` or `node` to define consensus participants